### PR TITLE
feat: Find Persona

### PR DESCRIPTION
**Pull request by ghost**
1. Added locally prettier and ignored in .gitignore
2. removed yarn from start script in package json
3. Added a custom environment in serverless to avoid creating .env file
4. Added find persona handler

### CODE REVIEW

**Comment by ghost** [comment]
Opiniones en el punto 1 y 3 por favor :) @jorgechavez77

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,6 @@
 node_modules
 yarn.lock
 .env
+
+// I prefer using prettier, but I am ignoring it for git in case you have a different configuration
+.prettierrc
```
Comment (jorgechavez77) [inline .gitignore]: Missing end of line here

File: package.json [hunk 2]
```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "yarn sls offline"
```
Comment (jorgechavez77) [inline package.json:8]: This requiers to install sls as global

File: serverless.yml [hunk 3]
```diff
@@ -1,8 +1,13 @@
 service: sls-data-service
 
+custom:
+  MONGO_URI: mongodb://127.0.0.1:27017/gorditos
```
Comment (jorgechavez77) [inline serverless.yml:4]: Doea this force us to install mongo locally?

**Review by jorgechavez77 — CHANGES_REQUESTED** [review]
Please review comments below

File: .gitignore (hunk 1 above)
Comment (ghost) [reply .gitignore]: ok

File: package.json (hunk 2 above)
Comment (ghost) [reply package.json:8]: sls is installed as a dev dependency. We should be fine.

File: serverless.yml (hunk 3 above)
Comment (ghost) [reply serverless.yml:4]: I am not sure here. I was told we were supposed to use a local mongo db.
How can I get a connection string for the correct database?

File: package.json (hunk 2 above)
Comment (jorgechavez77) [reply package.json:8]: In order to use `sls` you need to install sls as global. So if you use `yarn sls` you do not have to

File: serverless.yml (hunk 3 above)
Comment (jorgechavez77) [reply serverless.yml:4]: You can set your env before the `sls offline` instead of putting them in the yml

**Review by jorgechavez77 — APPROVED** [review]

File: package.json (hunk 2 above)
Comment (ghost) [reply package.json:8]: https://stackoverflow.com/questions/44420381/how-does-npm-scripts-prioritise-local-dependency-over-global-ones

Comment (jorgechavez77) [reply package.json:8]: Yes

**Pull request merged**